external-crates/move/crates/move-compiler/src/cfgir/locals/mod.rs

```diff
@@ -219,24 +219,39 @@ fn lvalue(context: &mut Context, sp!(loc, l_): &LValue) {
                             LocalState::MaybeUnavailable { .. } => "might contain",
                         };
                         let available = *available;
-                        let vstr = match display_var(v.value()) {
-                            DisplayVar::Tmp => panic!("ICE invalid assign tmp local"),
-                            DisplayVar::Orig(s) => s,
+                        match display_var(v.value()) {
+                            DisplayVar::Tmp => {
+                                let msg = format!(
+                                    "This expression {} a value without the '{}' ability \
+                                    which must be used",
```

Comment:
The only reason `MaybeUnavailable` occurs here is because we hoist the temp to the function's scope. So qualifying this with `verb` feels unhelpful (and maybe confusing)

As such, I think we should just simplify this to 
```suggestion
                                    "This expression without the '{}' ability must be used",
```